.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## All Submissions:
+-------------------------------------
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] I have updated the documentation accordingly.
+* [ ] I have added tests to cover my changes.
```

Comment:
Can we call out integration and unit tests as separate items?

Sometimes it may be valid to answer "no" to some of these (example: no integration tests needed because the change is adds documentation). It would be helpful to have a section to let the PR owner explain why they answered "no"

Reply:
> Can we call out integration and unit tests as separate items?
> 
> Sometimes it may be valid to answer "no" to some of these (example: no integration tests needed because the change is adds documentation). It would be helpful to have a section to let the PR owner explain why they answered "no"

those are fantastic suggestions, and thank you. It's totally fine if the answer is no or N/A. The goal is for these answer's to surface areas of needed focus for the PR reviewer.